gcp-deployer/deploy/ssh.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deploy
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/golang/glog"
+)
+
+const (
+	MachineControllerSshKeySecret = "machine-controller-sshkeys"
+
+	// Arbitrary name used for SSH.
+	SshUser                = "clusterapi"
+	SshUserFile            = "clusterapi-user"
+	SshKeyFilePrivate      = "clusterapi-key"
+	SshKeyFilePublic       = SshKeyFilePrivate + ".pub"
+)
+
+func createSshKeyPairs() error {
+	if fileExists(SshUserFile) || fileExists(SshKeyFilePrivate) || fileExists (SshKeyFilePublic) {
+		if !fileExists(SshUserFile) || !fileExists(SshKeyFilePrivate) || !fileExists (SshKeyFilePublic) {
+			return fmt.Errorf(
+				"A incomplete set of ssh files exist and cannot be reused. Please remove all files to regenerate ssh keys: %v, %v, %v",
+				SshUserFile,
+				SshKeyFilePrivate,
+				SshKeyFilePublic)
+		} else {
+			glog.Info("Re-using existing ssh files.\n")
```

Comment:
Is this log statement accurate? it seems like the key files and user file are always recreated below on line 51 - 61

Reply:
Fixed